_posts/2023-12-13-infinispan-vector-search.adoc

```diff
@@ -0,0 +1,92 @@
+---
+layout: blog
+title: Infinispan kNN Vector Search
+permalink: /blog/:year/:month/:day/infinispan-vect
+date: '2022-07-18T19:37:00.000-00:00'
+author: fax4ever
+tags: [ "opentelemetry", "tracing" ]
+---
+
+= Infinispan kNN Vector Search
+
+Dear Infinispan community,
+
+With the Infinispan 15 development release 06, we started to expose vector search capabilities using the Infinispan indexed queries.
```

Comment:
I would use "present perfect" everywhere instead of simple past. 
"We have started to"